build.psm1

```diff
@@ -920,7 +922,24 @@ function Start-PSPester {
         }
         else
         {
-            & $powershell -noprofile -c $Command
+            if ($PassThru.IsPresent)
+            {
+                $passThruFile = [System.IO.Path]::GetTempFileName()
+                try 
+                {
+                    $Command += "|Export-Clixml -Path '$passThruFile' -Force"
+                    Start-NativeExecution -sb {& $powershell -noprofile -c $Command} | ForEach-Object { Write-Host $_}
```

Comment:
Only the PassThru object is captured by Export-Clixml... everything else is done by a write host, but when we run another PowerShell process that gets turned into an output.  Thus, the complexity.